docs/usage/conservation.ipynb

```diff
@@ -241,9 +241,6 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
```

Comment:
Could you undo the changes to the notebooks that you did not intend to edit in this PR?